static/panes/compiler.ts

```diff
@@ -1727,6 +1750,10 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
                     if (result?.result) {
                         this.handlePopularArgumentsResult(result.result);
                     }
+                })
+                .catch(error => {
+                    // Log the error but don't show to user - popular arguments are optional
+                    console.warn('Failed to fetch popular arguments:', error);
```

Comment:
Is this a fetch/http error? It should maybe be propagated to Sentry (but maybe that's what we're trying to prevent here?)

Reply:
yes, and yes. Went back and forth on this and decided ultimately, if we can't fetch arguments that's "ok". Sentry logs filled with people on bad internet connections (it seems) getting HTTP cancel events etc